scripts/update-collector-version.sh

```diff
@@ -1,64 +1,43 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
I generally always use this, b/c it ensures script will fail fast and not end up doing weird stuff after something unexpected occurs